src/po/fi.po

```diff
@@ -4920,14 +4872,14 @@ msgid "E301: Oops, lost the swap file!!!"
 msgstr "E301: Hups, swap-tiedosto hävisi!"
 
 msgid "E302: Could not rename swap file"
-msgstr "E302: Swap-tiedoston uudellennimeys ei onnistu"
+msgstr "E302: Swap-tiedoston uudellennimeäminen ei onnistu"
 
 #, c-format
 msgid "E303: Unable to open swap file for \"%s\", recovery impossible"
 msgstr "E303: Swap-tiedostoa %s ei voi avata, palautus ei onnistu"
 
 msgid "E304: ml_upd_block0(): Didn't get block 0??"
-msgstr "E304: ml_upd_block0(): Lohko 0:aa ei saatu?"
```

Comment:
This is not how to inflect numbers written in digits in Finnish, see http://www.kielitoimistonohjepankki.fi/ohje/268